frontend/src/slices/styleDataSlice.ts

```diff
@@ -14,7 +23,8 @@ export const validStyleFunctions = {
     return check.color === strColor.toLowerCase();
   },
   textFont: (fontType) => {
-    return fontType;
+    // return fontType;
```

Comment:
This can be removed.

Reply:
the comment was originally documentation for what was previously returned but I made that clearer now.